tests/scripts/components-configuration.sh

```diff
@@ -0,0 +1,152 @@
+# components-configuration.sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+# This file contains test components that are executed by all.sh
+
+################################################################
+#### Configuration Testing
+################################################################
+
+component_tf_psa_crypto_test_default_out_of_box () {
+    msg "build: make, default config (out-of-box)" # ~1min
```

Comment:
I think it should be:
```suggestion
    msg "build: cmake, default config (out-of-box)" # ~1min
```
but actually I have a question about this: specifying `cmake` made sense in the main repo because there you can either build with `make` or `cmake`, but in `tf-psa-crypto` there's only 1 option and that's `cmake`. Can't we just fix all the message strings in this file by removing cmake?

Reply:
Yeah, I think that is a better solution since it is just not necessary in a single build system project.

Reply:
Yes that makes sense, @Harry-Ramsey please do that clean-up.